README.md

```diff
@@ -32,6 +32,33 @@ cd tattoo
 ./run.sh -- -t paper-input -t paper-icon-button --verbose
 ```
 
+***Test repos with a config.***
+
+Create a json branch config file with keys of component names and values of the
+form {org: string, repo: string, ref: string} like below:
+
+
+`branches.json`:
+```
+{
+  "polymer" : {
+    "org": "Polymer",
```

Comment:
does the config need to be plait like this? Could the user just specify an bower-like identifier like `"polymer": Polymer/polymer#lazy-register-extends"`